src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -312,6 +342,8 @@ protected function prepareForDenormalization($data)
      * @param array|bool $allowedAttributes
      *
      * @return \ReflectionMethod|null
+     *
+     * @deprecated
      */
     protected function getConstructor(array &$data, string $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes)
```

Comment:
Since it should be put in the context now, I made this function deprecated for now, should I add some message onto it ?

Reply:
Yes I think you should add a call to `trigger_error`

Reply:
Fixed